src/docs/content/docs/api.md

```diff
@@ -58,18 +58,19 @@ This operation expects that a call was previously made to `/login` and that the
   * Expected query parameters:
       * *seedHost*: Host name or IP address of the added Cassandra
         clusters seed host.
+      * *jmxPort*: Custom JMX port.
   * Adds a new cluster to the service, and returns the newly added cluster object, if the operation was successful. If the cluster is already registered, the list of host will be updated to match the current topology.
   
   
 * **PUT     /cluster/{cluster_name}**
   * Expected query parameters:
       * *seedHost*: New host name or IP address used as Cassandra cluster seed.
+      * *jmxPort*: Custom JMX port.
   * Modifies a cluster's seed host. Comes in handy when the previous seed has left the cluster.
   
   
 * **DELETE  /cluster/{cluster_name}**
-  * Expected query parameters:
-      * *force* : Enforce deletion of the cluster even if there are active schedules and a repair run history (Optional)
+  * Expected query parameters: *None*
```

Comment:
You might need to rebase on top of master to get the `force` param back. It's a new addition from Reaper 2.0.